styles/Main.module.css

```diff
@@ -23,7 +23,7 @@
 }
 
 .colorB {
-    @apply text-[#1d4ed8];
```

Comment:
please change the class name, or move the color to a variable as described here https://tailwindcss.com/docs/customizing-colors#using-custom-colors